ai/openai_test.go

```diff
@@ -13,10 +13,11 @@ func TestDecodeChatCompletionRequest(t *testing.T) {
 		data string
 	}
 	tests := []struct {
-		name          string
-		args          args
-		wantReq       openai.ChatCompletionRequest
-		wantErrString string
+		name             string
+		args             args
+		wantReq          openai.ChatCompletionRequest
+		wantAgentContext map[string]string
```

Comment:
Is it possible to define `wantAgentContext` as type `map[string]any`?

Reply:
https://github.com/yomorun/yomo/blob/9cc6e536ea44237c945a5e9a8aad3148013415d9/core/metadata/metadata.go#L18

The type of `metadata` limits this.